src/views/wallet/importsView/ImportsView.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useMemo } from 'react'
+
+import { Tabs, Row } from 'antd'
+
+import BackLink from '../../../components/uielements/backLink'
+import Label from '../../../components/uielements/label'
+import ImportPhrase from '../../../components/wallet/ImportPhrase'
+import { ImportsViewWrapper } from './ImportsView.style'
+
+enum TabKey {
+  PHRASE = 'phrase',
+  KEYSTORE = 'keystore'
+}
+
+type Tab = {
+  key: TabKey
+  label: string
+  content: React.ReactNode
+}
+
+const ImportsView: React.FC = (): JSX.Element => {
+  const items = useMemo(() => [{ key: TabKey.PHRASE, label: 'Phrase', content: <ImportPhrase /> }] as Tab[], [])
```

Comment:
its better not to typecast interfaces manually.
in case of typecast with `as` you allowed to do this thing
```ts
const items = useMemo(() => [{ label: 'Phrase', content: <ImportPhrase /> }] as Tab[], []) // hard to notice when you debugging anything that there is missed key value
// or even this
const items = useMemo(() => [{}] as Tab[], []) // more obvious but anyway
```
better to use explicit way without casting
```ts
const items: Tab[] = useMemo(() => [{ key: TabKey.PHRASE, label: 'Phrase', content: <ImportPhrase /> }], [])
```